pkg/ingester/active_series_test.go

```diff
@@ -8,27 +8,33 @@ import (
 	"sync"
 	"testing"
 	"time"
+	"unsafe"
 
 	"github.com/prometheus/prometheus/model/labels"
 	"github.com/stretchr/testify/assert"
 )
 
 func copyFn(l labels.Labels) labels.Labels { return l }
 
+func FromLabelAdaptersToLabels(ls []labels.Label) labels.Labels {
+	return *(*labels.Labels)(unsafe.Pointer(&ls))
+}
```

Comment:
Is this function still necessary? Can we use the same function in `cortexpb`?

Reply:
you may right. But I have tried this early, it shows struct formart errors because cortexpb is used for an adaptor label. If we use this function, many test case's original label struct will be transfer to adaptor again. It seems weird.

Reply:
Oh I see. So they are different types actually. Can we rename the method to something else? And if it is not necessary let's make the method private

Reply:
sounds good.